### PR TITLE
[WIP] Add builtin digdag parameters

### PR DESCRIPTION
**Pull request by naritta**
I added 2 types of parameters.
1. New parameters to be 

- Project Name ->`_digdag_project_name `
- Workflow ID -> `_digdag_workflow_id `
- Workflow Name -> `_digdag_workflow_name `
- Revision -> `_digdag_revision_id `

2. Add `_digdag_` in addition to current existing parameters for backward compatibility like:
`session_uuid` -> `_digdag_session_uuid `

### CODE REVIEW

File: digdag-tests/src/test/java/acceptance/RuntimeParamsIT.java [hunk 1]
```diff
@@ -198,4 +205,55 @@ public void testParamWithSchedule()
         assertEquals("+params2_sch+t_task_name", result.get("task_name", String.class));
         assertEquals("retry01", result.get("retry_attempt_name", String.class));
     }
+
+    @Test
+    public void testbuildinParams()
```
Comment (yoyama) [inline digdag-tests/src/test/java/acceptance/RuntimeParamsIT.java]: I guess this test should be moved to unit test.
And you should check these new parameter is really referable from workflow.
It is better to push a test workflow and run and check workflow refer to new variable correctly.
I guess you may be able refer to testParamWithSchedule().

**Comment by hiroyuki-sato** [comment]
Hello, @naritta 

Could you tell me why these parameters append prefix `_digdag_`?
It just prepares for explaining to other people.

Best regards.

**Comment by naritta** [comment]
@hiroyuki-sato Sorry to be late for response, I'll add documentation for that soon after design is defined.
Thanks.

**Comment by yoyama** [comment]
Close because no activity long time.

**Pull request closed without merging**